NOTICE

```diff
@@ -39,6 +39,8 @@ under the licensing terms detailed in LICENSE:
 * bnbarak <bn.barak@gmail.com>
 * Colin Eberhardt <colin.eberhardt@gmail.com>
 * Ryan Pivovar <ryanpivovar@gmail.com>
+* Joe Pea <trusktr@gmail.com>
+* Joe Pea <joe@trusktr.io>
```

Comment:
could you leave only one email? Which linked to Github profile only

Reply:
```suggestion
* Joe Pea <trusktr@gmail.com>
```

Reply:
```suggestion
* Joe Pea <trusktr@gmail.com>
```

Reply:
cc @trusktr

Reply:
I thought I'd put both in case I commit with one or the other.

Reply:
But I suppose I can try to fix my systems to have one email.

Reply:
Ok, I removed one.

Reply:
hmm, it seems wrong mail. Try `<joe@trusktr.io>`

Reply:
Huh, interesting. I verified I have both emails in my GitHub account. Updated to the other one.